pyshelf/search/metadata.py

```diff
@@ -0,0 +1,37 @@
+from elasticsearch_dsl import String, Nested, Boolean, DocType
+
+
+class Metadata(DocType):
+    items = Nested(
+        properties={
+            "name": String(),
+            "value": String(),
+            "immutable": Boolean()
+        }
+    )
+
+    class Meta:
+        index="metadata"
+
+    def update_all(self, metadata):
+        """
+            Updates all metadata related to an artifact.
+
+            Args
+                item(string): value to set the metdata item to.
+        """
+        self.items = metadata.values()
+        self.save()
+
+    def update_item(self, item):
```

Comment:
Ya know.  I don't think this will get used.  Can you think of an instance this will get used?  If not, can we just remove this?